graylog2-server/src/main/java/org/graylog2/indexer/IndexMapping.java

```diff
@@ -37,8 +37,7 @@
         return ImmutableMap.of(
                 "properties", partFieldProperties(analyzer),
                 "dynamic_templates", partDefaultAllInDynamicTemplate(),
-                // Compress source field
-                "_source", enabledAndCompressed(),
+                "_source", enabled(),
```

Comment:
Is the `_source` field compressed by default now? Or what is the reason to remove the compressions part?